src/CommunityToolkit.Maui.Core/Essentials/SpeechToText/SpeechToTextImplementation.windows.cs

```diff
@@ -86,7 +86,7 @@ async Task<string> ListenOnline(CultureInfo culture, IProgress<string>? recognit
 		await using (cancellationToken.Register(async () =>
 		{
 			await StopRecording();
-			speechRecognitionTaskCompletionSource.SetCanceled();
+			speechRecognitionTaskCompletionSource.TrySetCanceled();
```

Comment:
This's a breaking change because we'll change the behavior. The `SetCanceled` method will throw an exception if we try to set the value more than once, the `TrySetCanceled` will fail silently.

Reply:
we do not expect an exception. this was a mistake, and now each time on windows we receive an exception, that result is already set

Reply:
Anyway it will be a breaking change

Reply:
lets keep it

Reply:
Ok, just remember to release this change in a major version